Essentials/src/com/earth2me/essentials/commands/EssentialsLoopCommand.java

```diff
@@ -100,19 +108,21 @@ protected void loopOnlinePlayers(final Server server, final CommandSource sender
                         continue;
                     }
                     foundUser = true;
-                    updatePlayer(server, sender, player, commandArgs);
+                    userConsumer.accept(player);
                 }
             }
             if (!foundUser) {
                 throw new PlayerNotFoundException();
             }
         } else {
             final User player = getPlayer(server, sender, searchTerm);
-            updatePlayer(server, sender, player, commandArgs);
+            userConsumer.accept(player);
         }
     }
 
-    protected abstract void updatePlayer(Server server, CommandSource sender, User user, String[] args) throws NotEnoughArgumentsException, PlayerExemptException, ChargeException, MaxMoneyException;
+    protected void updatePlayer(Server server, CommandSource sender, User user, String[] args) throws NotEnoughArgumentsException, PlayerExemptException, ChargeException, MaxMoneyException {
```

Comment:
I'd prefer keeping this abstract, it's still used in most places, and the few cases where it's unused we can implement the empty method.